services/src/main/java/org/keycloak/broker/kubernetes/KubernetesIdentityProviderFactory.java

```diff
@@ -5,6 +5,7 @@
 import org.keycloak.broker.provider.AbstractIdentityProviderFactory;
 import org.keycloak.common.Profile;
 import org.keycloak.models.IdentityProviderModel;
+import org.keycloak.models.IdentityProviderShowInAccountConsole;
```

Comment:
There's unused imports that can be removed.

Reply:
Thanks for spotting this! Fixed.